templates/coderd.yaml

```diff
@@ -9,12 +9,9 @@ metadata:
     coder.deployment: {{ include "coder.serviceName" . }}
   name: {{ include "coder.serviceName" . }}
   namespace: {{ .Release.Namespace | quote }}
-  annotations:
-  {{- range $key, $value := fromYaml (include "movedValue" (dict "Values" .Values "Key" "services.annotations")) }}
-    {{ $key }}: {{ $value | quote }}
-  {{- end }}
+  annotations: {{- toYaml .Values.services.annotations | nindent 4 }}
```

Comment:
I think this needs to be on it's own line and without the `-` prefix

Reply:
the `nindent` is supposed to add a newline first, I think. what does the `-` prefix do?

Reply:
Actually in that case this is probably fine as long as you're consistent with this style (`-` prefix and `nindent`)

The `-` prefix or suffix after the `{{` or before the `}}` means "eat all whitespace before or after this block".

Reply:
nice, good to know, thanks!